toolbox/linux/index.rst

```diff
@@ -0,0 +1,32 @@
+Linux
+=====
+
+`Linux <https://en.wikipedia.org/wiki/Linux>`_ is an `Operating System
+<https://en.wikipedia.org/wiki/Operating_system>`_, like Windows or Mac OS X
+[#fn-kernel]_. Unlike those operating systems, Linux is open source, and
+completely free of charge. While you may expect something that costs no money
+and is developed by a team of "volunteers"[#fn-vol]_ to be low quality, Linux
+is actually very good.
+
+If you follow through this subsection, you should be at the point where you have
+installed Linux on your computer or a virtual machine. This is a prerequisite
+for a number of things in the "Hacker's Toolbox."
```

Comment:
> This is a prerequisite for a number of things in the "Hacker's Toolbox."

I don't think this is actually true. You really just need a unix system, so people with Macs + homebrew/macports should be fine. I bet you could even get away with BSD :)